src/js/components/Button/__tests__/Button-kind-test.js

```diff
@@ -0,0 +1,259 @@
+import React from 'react';
+
+import 'jest-styled-components';
+import 'jest-axe/extend-expect';
+import 'regenerator-runtime/runtime';
+
+import { axe } from 'jest-axe';
+import { cleanup, fireEvent, render } from '@testing-library/react';
+import { Grommet, Button } from '../..';
+import { customTheme } from '../Button-kind-theme';
```

Comment:
The filename "Button-kind-theme.js" should be renamed to lower case, we usually use title case for components. 
Also, I think we should change the name of the theme (customTheme) to match the name of the file, so maybe a name without hyphens.